src/components/Project/Releases/index.js

```diff
@@ -171,6 +178,22 @@ class ReleaseView extends React.Component {
     }, 1000)
   }  
 
+  renderDetailsLine() {
+    let timestamp = moment(new Date(this.props.release.created)).format("ddd, MMM Do, YYYY HH:mm:ss")
+    let timezone = moment.tz(jstz.determine().name()).format('z')
+
+    let author = "Automated Deployment"
```

Comment:
We should change this to Continuous Deployment to be consistent with the project settings property.